tailwind.config.mjs

```diff
@@ -104,6 +104,7 @@ export default {
         },
       },
       screens: {
+        coarse: { raw: '(pointer: coarse)' },
```

Comment:
Perhaps "touch" would be a better name, since that's what we mean 99% of the time. `pointer: coarse` is a result of the spec authors trying to be very general about the meaning (eg. to cover all "low-resolution" input methods rather than just fingers).

Reply:
I like your idea here! I will make this change.